src/Native/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -1358,697 +1358,3 @@ REDHAWK_PALEXPORT _Ret_maybenull_ void* REDHAWK_PALAPI PalSetWerDataBuffer(_In_
     static void* pBuffer;
     return InterlockedExchangePointer(&pBuffer, pNewBuffer);
 }
-
-#ifndef RUNTIME_SERVICES_ONLY
-
-static bool g_SeLockMemoryPrivilegeAcquired = false;
-
-bool InitLargePagesPrivilege()
-{
-    TOKEN_PRIVILEGES tp;
-    LUID luid;
-    if (!LookupPrivilegeValueW(nullptr, SE_LOCK_MEMORY_NAME, &luid))
-    {
-        return false;
-    }
-
-    tp.PrivilegeCount = 1;
-    tp.Privileges[0].Luid = luid;
-    tp.Privileges[0].Attributes = SE_PRIVILEGE_ENABLED;
-
-    HANDLE token;
-    if (!OpenProcessToken(::GetCurrentProcess(), TOKEN_ADJUST_PRIVILEGES, &token))
-    {
-        return false;
-    }
-
-    BOOL retVal = AdjustTokenPrivileges(token, FALSE, &tp, 0, nullptr, 0);
-    DWORD gls = GetLastError();
-    CloseHandle(token);
-
-    if (!retVal)
-    {
-        return false;
-    }
-
-    if (gls != 0)
-    {
-        return false;
-    }
-
-    return true;
-}
-
-static AffinitySet g_processAffinitySet;
-
-class GroupProcNo
-{
-    uint16_t m_groupProc;
-
-public:
-
-    static const uint16_t NoGroup = 0x3ff;
-
-    GroupProcNo(uint16_t groupProc) : m_groupProc(groupProc)
-    {
-    }
-
-    GroupProcNo(uint16_t group, uint16_t procIndex) : m_groupProc((group << 6) | procIndex)
-    {
-        assert(group <= 0x3ff);
-        assert(procIndex <= 0x3f);
-    }
-
-    uint16_t GetGroup() { return m_groupProc >> 6; }
-    uint16_t GetProcIndex() { return m_groupProc & 0x3f; }
-    uint16_t GetCombinedValue() { return m_groupProc; }
-};
-
-static LARGE_INTEGER g_performanceFrequency;
-
-#ifdef PROJECTN
-static bool g_roInitialized;
-#endif
-
-// Initialize the interface implementation
-// Return:
-//  true if it has succeeded, false if it has failed
-bool GCToOSInterface::Initialize()
-{
-    if (!::QueryPerformanceFrequency(&g_performanceFrequency))
-    {
-        return false;
-    }
-
-#ifdef PROJECTN
-    // TODO: Remove the RoInitialize call when we implement non-WinRT framework for classic apps
```

Comment:
Am i lucky and this todo is done or will i have to do something about this?

Reply:
This is one more thing that would need to be dealt with if the GC updates are ingested in ProjectN. Your change is fine.